app/helpers/prop.scale.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Creates a scale validation function for scale value props.
+ *
+ * @param {Array.<Number>} validNumbers - An Array of valid scale numbers.
+ * @param {Number} [defaultValue = 500] - The default value for this prop.
+ *
+ * @returns {Object}
+ */
+export default function(validNumbers, defaultValue = 500) {
```

Comment:
- [ ] I don't think we should use a defined default value as this would lead to an error if no default is set and the valid numbers don't contain '500'.
Another option would be to automatically use the first entry if no default is set.